maps/map_files/USS_Almayer/USS_Almayer.dmm

```diff
@@ -18063,9 +18063,9 @@
 	},
 /area/almayer/medical/lower_medical_lobby)
 "beZ" = (
-/obj/structure/platform{
+/obj/structure/platform_decoration{
 	dir = 8;
-	icon_state = "platform_deco"
+	tag = "icon-platform_deco (EAST)"
```

Comment:
sanitize tags when you can, we do not use them anymore. Frankly we should regex the whole damn repo and remove 'em. IIRC only foliage code still has support for var/tag and uses it to randomize sprites.